### ISSUE TITLE
Wheels on PyPI for CPython 3.12?

### ISSUE THREAD

**Issue by paugier** [pyFFTW/pyFFTW#367]
I would need wheels on PyPI for Python 3.12. What is the status of building pyfftw for 3.12?

I guess #363 should be merged before trying to build pyfftw for Python 3.12 and updating the Github Actions configuration and pyproject.toml.

**Comment by hgomersall** [comment]
Yeah, it should be  pretty simple to update for 3.12. Please feel free to initiate a PR for this.

**Comment by hgomersall** [comment]
I've just merged #363

**Comment by hgomersall** [comment]
It seems we're now limited to Python >=3.9. This seems to be inadvertent but was a result of #365

**Comment by hgomersall** [comment]
Actually, it suspect it's numpy dropping support. We should remove 3.8 at the same time.

**Comment by paugier** [comment]
Done.
